test/test_dht.cpp

```diff
@@ -136,6 +136,57 @@ void lazy_from_entry(entry const& e, bdecode_node& l)
 	TEST_CHECK(ret == 0);
 }
 
+void send_dht_request2(node& node, char const* msg, udp::endpoint const& ep
```

Comment:
any suggestion? 
I think the original send_dht_request has too many parameters, It's not convenient in some cases from my point of view, I prefer to use this one everywhere. But I don't want change it in this patch. So do you think it is acceptable if I replace the original one to this, and change this function name to send_dht_reuqest ?